Traceback (most recent call last):
  File "/mnt/beegfs/homes/simondn/RashomonActiveLearning/Code/RunSimulation.py", line 42, in <module>
    SimulationResults = OneIterationFunction(DataFileInput = args.Data,
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/beegfs/homes/simondn/RashomonActiveLearning/Code/utils/Main/OneIterationFunction.py", line 77, in OneIterationFunction
    ErrorVec, SelectedObservationHistory = LearningProcedure(df_Train = df_Train, 
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/beegfs/homes/simondn/RashomonActiveLearning/Code/utils/Main/LearningProcedure.py", line 57, in LearningProcedure
    QueryObservationIndex = SelectorType(**SelectorArgsFiltered)
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/beegfs/homes/simondn/RashomonActiveLearning/Code/utils/Selector/TreeEnsembleQBCFunction.py", line 44, in TreeEnsembleQBCFunction
    PredictedValues = [Model[i].predict(df_Candidate) for i in LowestErrorIndices]
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/beegfs/homes/simondn/RashomonActiveLearning/Code/utils/Selector/TreeEnsembleQBCFunction.py", line 44, in <listcomp>
    PredictedValues = [Model[i].predict(df_Candidate) for i in LowestErrorIndices]
                       ~~~~~^^^
  File "/mnt/beegfs/homes/simondn/RashomonActiveLearning/Code/treeFarms/treefarms/model/treefarms.py", line 135, in __getitem__
    return self.model_set.__getitem__(idx)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/homes/simondn/.local/lib/python3.11/site-packages/treefarms/model/model_set.py", line 63, in __getitem__
    return self.get_tree_at_idx(key)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/homes/simondn/.local/lib/python3.11/site-packages/treefarms/model/model_set.py", line 66, in get_tree_at_idx
    return TreeClassifier(self.get_tree_at_idx_raw(idx))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/homes/simondn/.local/lib/python3.11/site-packages/treefarms/model/model_set.py", line 100, in get_tree_at_idx_raw
    raise "Index exceeds total stored trees"
TypeError: exceptions must derive from BaseException
